sdks/typescript/client-typescript-eventsource/README.md

```diff
@@ -306,30 +317,27 @@ export enum Readyness {
 ```
 ## Rollout Strategies
 
-FeatureHub at its core now supports _server side_ evaluation of complex rollout strategies, both custom ones
-that are applied to individual feature values in a specific environment and shared ones across multiple environments
-in an application. Exposing that level fo configurability via a UI is going to take some time to get right,
-so rather than block until it is done, Milestone 1.0's goal is to expose the percentage based rollout functionality
-for you to start using straight away.
+Starting from version 1.1.0 FeatureHub supports _server side_ evaluation of complex rollout strategies
+that are applied to individual feature values in a specific environment. This includes support of generic rules, e.g. per user key, country, device type as well as percentage splits rules and custom rules that you can create according to your application needs. 
+
+For more details on rollout strategies, targeting rules and feature experiments see the [core documentation](https://docs.featurehub.io/#_rollout_strategies_and_targeting_rules).
+
+We are actively working on supporting client side evaluation of
+strategies in the future releases as this scales better when you have 10000+ consumers. 
 
-Future Milestones will expose more of the functionality via the UI and will support client side evaluation of
-strategies as this scales better when you have 10000+ consumers. For more details on how
-experiments work with Rollout Strategies, see the [core documentation](https://docs.featurehub.io).
+#### Coding for rollout strategies 
+There are several generic attribute rules we track specifically: `user key`, `country`, `device` and `platform`. However, if those do not satisfy your requirements you also have an ability to attach a custom rule. Custom rules can be created as following types: `string`, `number`, `boolean`, `date`, `date-time`, `semantic-version`, `ip-address` 
 
-#### Coding for Rollout strategies 
-To provide this ability for the strategy engine to know how to apply the strategies, you need to provide it
-information. There are five things we track specifically: user key, session key, country, device and platform and
-over time will be able to provide more intelligence over, but you can attach anything you like, both individual
-attributes and arrays of attributes. 
+In order for the strategy engine to know how to apply the strategies rules, you are required to provide the following:
```

Comment:
this implies that you are required to provide all the ones in your example

Reply:
sorry, don't get this one, I provide all of the possible ones